### PR TITLE
Update deprecations section for CLI

### PR DESCRIPTION
**Pull request by stevekinney**
@locks and I think L96 should possibly be reworded, but we're not sure how to do that without losing clarity. Other than that, this could potentially be merged pending review.

### CODE REVIEW

File: source/deprecations/index.md [hunk 1]
```diff
@@ -99,14 +99,19 @@ var App.ListView = Ember.View.extend(ListView);
 A more advanced solution is to use an initializer to register the plugin's
 views on the the application:
 
-```JavaScript
+```app/initializers/list-view.js
 // usage is {{view "list"}}
-Ember.Application.initializer({
-  name: 'list-view',
+
+function initialize(container, application) {
```
Comment (locks) [inline source/deprecations/index.md]: I think you still need to `export function initialize…` here

Comment (locks) [inline source/deprecations/index.md]: at least that's seems to be how it's generated

**Comment by locks** [comment]
:shipit:

File: source/deprecations/index.md [hunk 2]
```diff
@@ -99,14 +99,19 @@ var App.ListView = Ember.View.extend(ListView);
...
 views on the the application:
 
-```JavaScript
+```app/initializers/list-view.js
 // usage is {{view "list"}}
-Ember.Application.initializer({
-  name: 'list-view',
+
+export function initialize(container, application) {
   initialize: function(container, application) {
```
Comment (mixonic) [inline source/deprecations/index.md]: This new version seems incorrect. it has both the old property and the new function.

Comment (locks) [inline source/deprecations/index.md]: oops, well spotted

Comment (stevekinney) [inline source/deprecations/index.md]: Oh yea, duh.

File: source/deprecations/index.md [hunk 3]
```diff
@@ -99,14 +99,17 @@ var App.ListView = Ember.View.extend(ListView);
 A more advanced solution is to use an initializer to register the plugin's
 views on the the application:
 
-```JavaScript
+```app/initializers/list-view.js
 // usage is {{view "list"}}
-Ember.Application.initializer({
+
+export function initialize(container, application) {
+  container.register('view:list', ListView);
```
Comment (mixonic) [inline source/deprecations/index.md]: `ListView` should probably be imported.

Comment (stevekinney) [inline source/deprecations/index.md]: Done!

**Comment by trek** [comment]
Deprecations is moving out of guides (since it is version independent) and to a page on the website. See #70 and https://github.com/emberjs/website/pull/2076

**Comment by stevekinney** [comment]
So, I'll close this issue. Do you think I should open a PR on emberjs/website with the changes?

**Comment by trek** [comment]
Yes please.

**Pull request closed without merging**